lib/active_interaction/backports.rb

```diff
@@ -1,5 +1,7 @@
 # coding: utf-8
 
+require 'active_support/core_ext'
```

Comment:
Turns out ActiveSupport's core extensions aren't included when you require ActiveSupport.

``` rb
require 'active_support'
{}.deep_dup
# NoMethodError: undefined method `deep_dup' for {}:Hash

require 'active_support/core_ext'
{}.deep_dup
# => {}
```